python-sdk/example_dags/example_snowflake_partial_table_with_append.py

```diff
@@ -102,10 +102,12 @@ def example_snowflake_partial_table_with_append():
     extracted_data = extract_data(
         homes1=homes_data1,
         homes2=homes_data2,
-        output_table=Table(name="combined_homes_data"),
+        output_table=Table(name="combined_homes_data", temp=True),
```

Comment:
We should name this table uniquely so that concurrent run on CI doesn't fail it.

Reply:
changed and also made table name unique